controllers/azurecluster_controller.go

```diff
@@ -169,6 +180,7 @@ func (r *AzureClusterReconciler) reconcileNormal(ctx context.Context, clusterSco
 
 	// No errors, so mark us ready so the Cluster API Cluster Controller can pull it
 	azureCluster.Status.Ready = true
+	conditions.MarkTrue(azureCluster, infrav1.NetworkInfrastructureReadyCondition)
```

Comment:
is AzureCluster ready == infrav1.NetworkInfrastructureReadyCondition true?

Reply:
yes, should I rename it?

Reply:
FYI, the ready condition is already used for hosting the summary of all the conditions on an object